DigitalLearningSolutions.Web/Styles/myAccount/myAccount.scss

```diff
@@ -15,3 +15,25 @@
   justify-content: space-between;
   flex-wrap: wrap;
 }
+
+.account-switch-warning {
+  margin-bottom: 15px;
+}
+
+.account-switch-warning-msg {
+  background-color: #f7e178;

```

Comment:
Do we have scss colour variables for these values? Feels a bit 'magic number' like atm maybe (imho)?

Reply:
we do not have variables providing consistency. The colour is taken from the requirements.